examples/events/main.go

```diff
@@ -30,11 +30,14 @@ import (
 	"github.com/vmware/govmomi/vim25/types"
 )
 
-// example use: go run main.go -url $GOVC_URL -b 8h -f VmEvent
 func main() {
-	begin := flag.Duration("b", time.Hour, "Begin time") // default BeginTime is 1h ago
+	// example use against simulator: go run main.go -b 8h -f
+	// example use against vCenter with optional event filters:
+	// go run main.go -url $GOVMOMI_URL -insecure $GOVMOMI_INSECURE -b 8h -f VmEvent UserLoginSessionEvent
+	begin := flag.Duration("b", 10*time.Minute, "Begin time") // default BeginTime is 10min ago
 	end := flag.Duration("e", 0, "End time")
 	follow := flag.Bool("f", false, "Follow event stream")
+	flag.Parse()
```

Comment:
This is called already within `examples.Run`: https://github.com/vmware/govmomi/blob/c1c1989ea3ece09ecd01e0c327542146abebdb2b/examples/examples.go#L133

Reply:
Thx, but only with this change the flags actually worked for me...hm...

Reply:
Update: found the issue which is caused by an unhandled error in `example.go`. Will update PR to fix this.